src/ir/LocalGraph.cpp

```diff
@@ -103,12 +99,45 @@ struct Flower : public CFGWalker<Flower, Visitor<Flower>, Info> {
   }
 
   void flow(Function* func) {
+    // Minimalist block information needed
+    struct FlowBlock {
+      size_t lastTraversedIteration; // last Traversed Iteration
```

Comment:
please elaborate on how the iteration number helps us know if we saw a block in the current iteration